vscode/src/notebookTemplate.ts

```diff
@@ -99,9 +99,14 @@ import matplotlib.pyplot as plt
 import numpy as np
 from collections import Counter
 
-results = qsharp.run("RandomBit()", shots=1000)
+# Use save_events to suppress the automatic printing of events for every shot
+results = qsharp.run("RandomBit()", shots=1000, save_events=True)
```

Comment:
I think this might be related to a change you made @swernli , but it was printing 1000 DumpMachines now when running to just get the results, so had to change here to suppress that.

Reply:
The sample always did that, didn't it? There is an explicit `DumpMachine()` in the `RandomBit` operation that's being called. If it wasn't producing output before, that was a bug.